includes/payment-methods/class-wc-stripe-upe-payment-method.php

```diff
@@ -43,13 +43,30 @@ abstract class WC_Stripe_UPE_Payment_Method {
 	 */
 	protected $token_service;
 
+	/**
+	 * Array of currencies supported by this UPE method
+	 *
+	 * @var array
+	 */
+	protected $supported_currencies;
+
+	/**
+	 * Wether this UPE method is enabled
+	 *
+	 * @var bool
+	 */
+	protected $enabled;
+
 	/**
 	 * Create instance of payment method
 	 *
 	 * @param WC_Payments_Token_Service $token_service Instance of WC_Payments_Token_Service.
 	 */
 	public function __construct( $token_service ) {
-		//      $this->token_service = $token_service;
+		$main_settings       = get_option( 'woocommerce_stripe_settings' );
+		$enabled_upe_methods = $main_settings['upe_checkout_experience_accepted_payments'];
+		$this->enabled       = in_array( static::STRIPE_ID, $enabled_upe_methods );
```

Comment:
```suggestion
		$this->enabled       = in_array( static::STRIPE_ID, $enabled_upe_methods, true );
```
I get a phpcs yellow squiggly on this line moaning that you're not using a strict comparison here. Minor, but any reason not to?

Reply:
No reason not to make it strict. Will do. :+1: